webapp/appModules/data/records/components/recordsView.js

```diff
@@ -10,6 +10,7 @@ import TablePaginator from '../../../../commonComponents/table/tablePaginator'
 
 import Survey from '../../../../../common/survey/survey'
 import NodeDef from '../../../../../common/survey/nodeDef'
+import { canEditRecord } from '../../../../../common/auth/authManager'
```

Comment:
our convention is to import the entire object and call function from it. e.g. AuthManager.canEditRecord